.pipelines/ESRPSigning_core.json

```diff
@@ -177,6 +179,12 @@
                 "vcomp140_app.dll",
                 "vcruntime140_1_app.dll",
                 "vcruntime140_app.dll", 
+                "modules\\FileExplorerPreview\\Microsoft.Web.WebView2.Core.dll",
+                "modules\\FileExplorerPreview\\Microsoft.Web.WebView2.WinForms.dll",
+                "modules\\FileExplorerPreview\\Microsoft.Web.WebView2.Wpf.dll",
+                "modules\\FileExplorerPreview\\Newtonsoft.Json.dll",
```

Comment:
why are we using Newtson.JSON versus just built in?

Reply:
why: we shifted away from newtson for all other utilities

Reply:
Fixed